Do not override CFLAGS, which causes universal builds to fail.
Do not look for dependencies installed with Fink.
Do not use "echo -n" which is not portable.
--- configure.orig	2013-06-14 22:43:05.000000000 -0500
+++ configure	2013-12-09 04:11:22.000000000 -0600
@@ -11343,29 +11343,6 @@
 
 #AC_CANONICAL_HOST
 
-CFLAGS=
-default_fink_path=/sw
-case $host_os in
-     darwin*)
-	           	      { $as_echo "$as_me:${as_lineno-$LINENO}: checking for a fink installation at $default_fink_path" >&5
-$as_echo_n "checking for a fink installation at $default_fink_path... " >&6; }
-     	      if test -d $default_fink_path; then
-	      	 { $as_echo "$as_me:${as_lineno-$LINENO}: result: found it!" >&5
-$as_echo "found it!" >&6; }
-		 { $as_echo "$as_me:${as_lineno-$LINENO}: Adding -I$default_fink_path/include to CPPFLAGS" >&5
-$as_echo "$as_me: Adding -I$default_fink_path/include to CPPFLAGS" >&6;}
-	      	 CPPFLAGS="-I$default_fink_path/include $CPPFLAGS"
-		 { $as_echo "$as_me:${as_lineno-$LINENO}: Adding -L$default_fink_path/lib to LDFLAGS" >&5
-$as_echo "$as_me: Adding -L$default_fink_path/lib to LDFLAGS" >&6;}
-		 LDFLAGS="-L$default_fink_path/lib $LDFLAGS"
-	      else
-		{ $as_echo "$as_me:${as_lineno-$LINENO}: result: none" >&5
-$as_echo "none" >&6; }
-		{ $as_echo "$as_me:${as_lineno-$LINENO}: You may need to add set CPPFLAGS and LDFLAGS for gmp, etc." >&5
-$as_echo "$as_me: You may need to add set CPPFLAGS and LDFLAGS for gmp, etc." >&6;}
-	      fi
-	      ;;
-esac
 
 ############################
 # Configs for Windows DLLs.
@@ -16074,7 +16051,7 @@
 fi
 
 
-echo -ne "\n"
+echo
 echo "global build variables"
 echo "-----------------------------------------"
 echo `date`
@@ -16091,7 +16068,7 @@
 echo "AM_YFLAGS:        $AM_YFLAGS"
 echo "YFLAGS:           $YFLAGS"
 echo "-----------------------------------------"
-echo -ne "\n"
+echo
 
 
 
